File: UI/careconnect_mobile/lib/models/search_objects/participant_additional_data.dart
import 'package:json_annotation/json_annotation.dart';

part 'participant_additional_data.g.dart';

@JsonSerializable(explicitToJson: true)
class ParticipantAdditionalData {
  bool? isUserIncluded;
  bool? isWorkshopIncluded;
  bool? isAttendanceStatusIncluded;
  bool? isChildIncluded;

  ParticipantAdditionalData({
    this.isUserIncluded,
    this.isWorkshopIncluded,
    this.isAttendanceStatusIncluded,
    this.isChildIncluded,
  });

  factory ParticipantAdditionalData.fromJson(Map<String, dynamic> json) =>
      _$ParticipantAdditionalDataFromJson(json);

  Map<String, dynamic> toJson() => _$ParticipantAdditionalDataToJson(this);
}

File: CareConnect/CareConnect.Models/SearchObjects/ParticipantAdditionalData.cs
using CareConnect.Models.SearchObjects;

namespace CareConnect.Models.SearchObjects
{
    public class ParticipantAdditionalData : BaseAdditionalSearchRequestData
    {
        public bool? IsUserIncluded { get; set; }

        public bool? IsWorkshopIncluded { get; set; }

        public bool? IsAttendanceStatusIncluded { get; set; }

        public bool? IsChildIncluded { get; set; }  

    }
}